docs/installation.rst

```diff
@@ -15,6 +15,12 @@ You can also install Airflow with support for extra features like ``s3`` or ``po
 .. code-block:: bash
 
     pip install "apache-airflow[s3, postgres]"
+    
+After installation, you'll need to initialize the database before you can run tasks:
```

Comment:
Thank you for your contribution!

One recommendation: instead of adding this in the 'Getting Airflow' subsection, which is served to guide installation of the Airflow package; it would be great if you could add it to a new subsection (maybe call it 'Initiating Airflow Database') at the end of this doc.